frontend/.eslintrc.cjs

```diff
@@ -20,6 +20,8 @@ module.exports = {
         "import/first": "off",
         "no-multiple-empty-lines": ["error", { "max": 2, "maxBOF": 0, "maxEOF": 0 }],
         "@typescript-eslint/no-explicit-any": ["off"],
+        "no-alert": "off",
+        "max-len": ["error", { "code": 150 }]
```

Comment:
Was this an increase, decrease, or an addition?

Reply:
Increase from max 100 to 150 characters per line
Reason being that the linter wants arrow function to stay on the same line but gets pissed when it's too much chara to handle :)